sdk/translation/Azure.AI.Translation.Document/tests/TranslationOperationLiveTests.cs

```diff
@@ -259,7 +248,7 @@ public async Task WrongSourceRightTarget()
             DocumentTranslationClient client = GetClient();
 
             var input = new DocumentTranslationInput(source, target, "fr");
-            DocumentTranslationOperation operation = await client.StartTranslationAsync(input);
+            DocumentTranslationOperation operation = await client.TranslationAsync(input, waitForCompletion: false);
```

Comment:
Something I noticed here is that unless I set `waitForCompletion: false` I won't be able to access the `operation.Status` as the operation will be thrown and the object `operation` won't be initilized. 
Am I missing something?

Reply:
ok. never mind. people don't need the status. I just need to update the test :)